reflex-dom-plaid/Reflex/Dom/Plaid/Link.hs

```diff
@@ -131,17 +134,24 @@ activatePlaidLinkDialog cfg onResult = do
     pure ()
     )
 
-  handle <- jsg (t_ "Plaid") ^. js1 (t_ "create") o
+  o ^. jss (t_ "onEvent") (fun $ \_ _ _ -> do
+    )
+
+  handle <- ( jsg (t_ "Plaid") ^. js1 (t_ "create") o )
+    `catch` (\(e :: JSException) -> do
+      liftIO $ print . displayException $ e
```

Comment:
It would be nice to minimally return `Either err Foo` here so the caller can then handle any error. We'll want to take the error and actually log it so we know when theres a breakage in the api.

Reply:
fixed